### PR TITLE
input created

### PR DESCRIPTION
**Pull request by MariaAlba1306**
search box input created and added to the header

### CODE REVIEW

File: src/app/public/features/components/header/header.component.ts [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
 
 @Component({
   selector: 'app-header',
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.css'],
 })
-export class HeaderComponent implements OnInit {
-  constructor() {}
+export class HeaderComponent {
+  @Input() searchbox: any = '';
```
Comment (agandia9) [inline src/app/public/features/components/header/header.component.ts]: the type for this var should be 'string', no?

File: src/app/public/features/components/header/header.component.ts [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
 
-  ngOnInit(): void {}
+  inputvalue: any;
```
Comment (agandia9) [inline src/app/public/features/components/header/header.component.ts]: avoid `any` types

File: src/app/public/shared/input/input.component.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  templateUrl: './input.component.html',
+  styleUrls: ['./input.component.css'],
+})
+export class InputComponent implements OnInit {
+  constructor() {}
+  @Input() inputId = '';
+  @Input() control = new FormControl();
+  @Input() label = '';
+
+  ngOnInit(): void {}
```
Comment (agandia9) [inline src/app/public/shared/input/input.component.ts]: avoid empty ngoininit

File: src/app/public/features/components/header/header.component.ts (hunk 1 above)
Comment (MariaAlba1306) [reply src/app/public/features/components/header/header.component.ts]: yes, changed here in [PR WPA-button](https://github.com/MariaAlba1306/github/pull/5)

File: src/app/public/features/components/header/header.component.ts (hunk 2 above)
Comment (MariaAlba1306) [reply src/app/public/features/components/header/header.component.ts]: done in https://github.com/MariaAlba1306/github/pull/5

File: src/app/public/shared/input/input.component.ts (hunk 3 above)
Comment (MariaAlba1306) [reply src/app/public/shared/input/input.component.ts]: changed in https://github.com/MariaAlba1306/github/pull/5

**Pull request merged**